src/kudu/tserver/tablet_server_options.h

```diff
@@ -53,6 +53,7 @@ struct TabletServerOptions : public kudu::server::ServerBaseOptions {
   std::function<void(const consensus::ElectionResult&)> edcb;
   std::function<void(int64_t)> tacb;
   std::function<void(const consensus::OpId id)> norcb;
+  std::function<void()> ldcb;
```

Comment:
Why are we abbreviating these callbacks into acronyms?

Reply:
Mind adding the information you put into the commit message here, or in some header / interface file, for future people reading through this code?
Something like:
```
// Leader detected callback which fires when a follower joins the ring
// with no term change or no no-op received. We keep track if we
// also receive a no-op. And only in the case that the NoOpReceivedCallback has not already
// been fired, we fire Leader Detected Calback. We could fire Leader Detected Callback all the time, but we need
// to reconcile on mysql side if there needs to be 2 callbacks or 1 in future.
```

Reply:
we are planning to use these acronymns often in general usage. Although it might be a problem for upstream,
within mysql it should be ok. Its like the term DMP

Reply:
some of these callbacks will need a reduction soon as 2 is better than 3.